.github/workflows/release-nym-vpn-cli.yml

```diff
@@ -157,16 +180,69 @@ jobs:
 
       - name: Install Protoc
         uses: arduino/setup-protoc@v2
-        if: contains(matrix.os, 'macos') || contains(matrix.os, 'mac-m1')
+        if: contains(matrix.os, 'macos') || contains(matrix.os, 'mac-m1') || contains(matrix.os, 'windows')
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Build wireguard (Android)
+        if: contains(matrix.target, 'android')
+        run: ./wireguard/libwg/build-android.sh
+
+      - name: Download wintun.zip and winpcap.zip
+        if: contains(matrix.os, 'windows')
+        shell: bash
+        run: |
+          curl --output ${GITHUB_WORKSPACE}/wintun.zip https://www.wintun.net/builds/wintun-0.14.1.zip
+          curl --output ${GITHUB_WORKSPACE}/winpcap.zip https://www.winpcap.org/install/bin/WpdPack_4_1_2.zip
+
+      - name: Unzip wintun.zip and winpcap.zip
+        if: contains(matrix.os, 'windows')
+        shell: bash
+        run: |
+          unzip ${GITHUB_WORKSPACE}/wintun.zip
+          unzip ${GITHUB_WORKSPACE}/winpcap.zip
+
+      - name: Move wintun.dll and packet.lib to build directory
+        if: contains(matrix.os, 'windows')
+        shell: bash
+        run: |
+          mv ${GITHUB_WORKSPACE}/wintun/bin/amd64/wintun.dll ${GITHUB_WORKSPACE}/lib/
+          mv ${GITHUB_WORKSPACE}/WpdPack/Lib/x64/Packet.lib ${GITHUB_WORKSPACE}/lib/
+
+      - name: Checkout mullvad libs for Windows
+        if: contains(matrix.os, 'windows')
+        uses: actions/checkout@v4
+        with:
+          repository: nymtech/nym-vpn-mullvad-libs
+          ref: main
+          path: "nym-vpn-mullvad-libs"
+          submodules: true
+
+      - name: Build winfw.dll from mullvad
+        if: contains(matrix.os, 'windows')
+        shell: bash
+        run: |
+          cd ${GITHUB_WORKSPACE}/nym-vpn-mullvad-libs
+          ./build-windows-modules.sh
+
+      - name: Move winfw.dll to build directory
+        if: contains(matrix.os, 'windows')
+        shell: bash
+        run: |
+          mv ${GITHUB_WORKSPACE}/nym-vpn-mullvad-libs/windows/winfw/bin/x64-Debug ${GITHUB_WORKSPACE}/lib/
+
       - name: Build native
-        if: ${{ contains(matrix.target, 'native') }}
+        if: contains(matrix.target, 'native') && !contains(matrix.os, 'windows')
         env:
           RUSTFLAGS: "-L ${{ env.WG_GO_LIB_PATH }}"
         run: cargo build --release --verbose
 
+      - name: Build Windows
+        if: contains(matrix.os, 'windows')
+        env:
+          RUSTFLAGS: "-L ${{ env.WG_GO_LIB_PATH }}/x86_64-pc-windows-msvc -L ${{ env.WG_GO_LIB_PATH }}/ -Clink-args=/LIBPATH:${{ env.WG_GO_LIB_PATH }}/x64-Debug"
```

Comment:
Not in scope for this PR, but maybe we should export to env the `RUSTFLAGS` in the `Set env` step, so that we can add more stuff to it there depending on the platform and keep here the same `Build native` step for all

Reply:
A question for @octol as well